lib/models/expression.dart

```diff
@@ -166,47 +177,63 @@ class PhotoFormExpression {
   PhotoFormExpression({
     this.image,
     this.caption,
+    this.thumbnail300,
+    this.thumbnail600,
   });
 
   factory PhotoFormExpression.fromMap(Map<String, dynamic> json) {
     return PhotoFormExpression(
       image: json['image'],
       caption: json['caption'],
+      thumbnail300: json['thumbnail300'],
+      thumbnail600: json['thumbnail600'],
     );
   }
 
   @HiveField(0)
   String image;
   @HiveField(1)
   String caption;
+  @HiveField(2)
+  String thumbnail300;
+  @HiveField(3)
+  String thumbnail600;
 
   Map<String, dynamic> toMap() => <String, dynamic>{
         'image': image,
         'caption': caption,
+        'thumbnail300': thumbnail300,
+        'thumbnail600': thumbnail600,
       };
 }
 
 class EventFormExpression {
```

Comment:
Can we migrate this to freeze?

Reply:
I wanted to, but there are too many places where Expression is used to do it in this PR